python/alibiexplainer/alibiexplainer/__main__.py

```diff
@@ -16,25 +16,45 @@
 import argparse
 from alibiexplainer import AlibiExplainer
 from alibiexplainer.explainer import ExplainerMethod #pylint:disable=no-name-in-module
+from typing import Dict
+import dill
+import os
+import json
 
-DEFAULT_MODEL_NAME = "model"
+DEFAULT_EXPLAINER_NAME = "explainer"
+EXPLAINER_FILENAME = "explainer.dill"
+CONFIG_ENV = "ALIBI_CONFIGURATION"
 
 parser = argparse.ArgumentParser(parents=[kfserving.server.parser]) #pylint:disable=c-extension-no-member
-parser.add_argument('--model_name', default=DEFAULT_MODEL_NAME,
-                    help='The name that the model is served under.')
+parser.add_argument('--explainer_name', default=DEFAULT_EXPLAINER_NAME,
+                    help='The name of model explainer.')
 parser.add_argument('--predict_url', help='The URL for the model predict function', required=True)
-parser.add_argument('--method',
+parser.add_argument('--type',
                     type=ExplainerMethod, choices=list(ExplainerMethod), default="anchor_tabular",
-                    help='Explainer method')
-parser.add_argument('--training_data', help='The URL for the training data')
+                    help='Explainer method', required=True)
+parser.add_argument('--savedExplainerUri', help='The URL of a pretrained explainer')
```

Comment:
Should we call this storage_uri? We've been talking about this in KFServing in general, as a concept for a "mounted dir on a container"

Reply:
White box explainers will need saved model - would this not be `storage_uri`? Maybe `explainer_uri`? I'll change to get the default from env.

Reply:
I think the idea is that all components could access the saved model at an env variable at $STORAGE_URI, no matter what component. In fact, several artifacts in addition to saved model could be accessible at this URI .